orca-core/src/main/java/com/netflix/spinnaker/orca/pipeline/model/BuildInfo.kt

```diff
@@ -2,7 +2,7 @@ package com.netflix.spinnaker.orca.pipeline.model
 
 abstract class BuildInfo<A>(
   open val name: String?,
-  open val number: Int,
+  open val number: String?,
```

Comment:
Why nullable when it wasn't before?

Reply:
i believe in this case the constant `-1` was being sent to indicate null value

Reply:
Found [this place](https://github.com/spinnaker/orca/pull/4316/files#r1020612013) that treats <= 0 as missing....`String?` works for me.